packages/main/src/components/protected/topbar/components/BigModal.jsx

```diff
@@ -6,23 +6,26 @@ import Styled from "styled-components";
 import noImg from "../assets/images/avatar_vct.svg";
 import { NoResult } from "./SearchNotFound";
 import { plugins } from "../utils/topbar-api";
+import { FaCaretDown } from "react-icons/fa";
 
 export const BigModal = ({
   onClose,
   inputValue,
   result,
   isLoadingUp,
-  clearSearch
+  clearSearch,
+  searchType
 }) => {
   const [results, setResult] = useState(result);
   const [isLoading, setLoading] = useState(isLoadingUp);
+  const [searchFocus, setSearchFocus] = useState(searchType);
 
-  const pluginName = window.location.href;
-  const newName = pluginName.split("/");
+  // const pluginName = window.location.href;
+  // const newName = pluginName.split("/");
 
```

Comment:
same to this

Reply:
The code is not in use right now and will break the app.